spec-0005/index.md

```diff
@@ -25,6 +26,35 @@ intended use-cases, and pseudo-code illustrating its use.
 Discuss how this would be implemented.
 -->
 
+### Build nightly wheels
+
+By building and uploading wheel for your project you make it easier for projects
+that depend on you to test against your latest changes. They can then report problems
+they find.
+
+There are a few steps to implementing this for your project:
+1. Get access to the area that nightly wheels are uploaded to
+2. Setup a CI step that builds wheels for your project
+3. Setup a CI step that uploads wheels to https://anaconda.org/scientific-python-nightly-wheels/
+
+For step (1) contact XXX. Deposit the token you get as a [secret on your repository](https://docs.github.com/en/actions/security-guides/encrypted-secrets) named `UPLOAD_TOKEN`.
+
+The work for step (2) depends on your project. You are probably already doing this for your
```

Comment:
Actually, I am not sure if you should make this assumption. A lot of smaller project might just push their sdist tarball and call it a release. Maybe link to some external doc related to Python releases?

Reply:
I don't know what good advice to give to people about this is. Should we tell people to use https://cibuildwheel.readthedocs.io/en/stable/?